.github/workflows/ci.yml

```diff
@@ -16,14 +16,28 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-      
+
+    - name: Build panacea-core
```

Comment:
> Initially, I tried to utilize services in workflow, but in this case, the panacea must remain running in the container.
In panacea-core image.
I'm not sure if it's good to provide a script like this in a panacea-core image.
Therefore, i handle this task in java-sdk

Yes. I think it's okay to wrap the `panacea-core:master` image for now, as you did. Then, I think we can name this step more intuitive such as `Build/Run panacea-core` or `Set up panacea-core`. 

Also, as you said, it's not mandatory to use the `service` in Github Actions. The main reason to use the `service` is that it provides the `wait-for` mechanism that we can delay starting our tests until the service is fully initialized (ready).
But, in our case, it's not that important.


As a suggestion,
In near future, I think we can improve the panacea-core Dockerfile as below:
```Dockerfile
# skipped long stories

CMD ["/root/init_script.sh && panacea-core" ]
```
Then, in the panacea-java, we can define a `my_init_script.sh` and mount it to the panacea-core docker container:
```bash
docker run .... \
  -e MY_MNEMONIC="something" \
  -v my_init_script.sh:/root/init_script.sh \
  ghcr.io/medibloc/panacea-core:master
```
Then, we don't need any additional `Dockerfile`s in the panacea-java.
This is actually what the Postgres Docker does (The `Initialization scripts` chapter in https://hub.docker.com/_/postgres).

Reply:
Until then, I think we can move the `Dockerfile` from the project root to `src/test/resources/scripts/`, because the `Dockerfile` in the root is usually an image that contains something for running the project itself (Java SDK itself in our case).

Reply:
Ah. also.
As everything that the `Dockerfile` does is just running the script, I think we move all init logic in the `simple-init-start.sh` into the `Dockerfile`.

Reply:
Thank you for a good idea.
This time, I will move `Dockerfile` to `src/test/resources/scripts`.
Next time, I will improve to delete the `Dockerfile`

Reply:
When I checked it, I think the -v option mounts folder.
[docker-entrypoint.sh](https://github.com/docker-library/postgres/blob/517c64f87e6661366b415df3f2273c76cea428b0/13/buster/docker-entrypoint.sh)

It would be a good idea to modify the panacea-core later to run after checking if there is `*.sh` in a particular folder like the script above.

Reply:
Yeah. Postgres does it like that, but their approach is quite complicated. We can make it much simpler. Let's do it later :)

Reply:
OK